pkg/audit/reporter.go

```diff
@@ -0,0 +1,97 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package audit
+
+import (
+	"context"
+
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/provider"
+	dbx "storj.io/storj/pkg/statdb/dbx"
+	proto "storj.io/storj/pkg/statdb/proto"
+	"storj.io/storj/pkg/statdb/sdbclient"
+)
+
+type reporter interface {
+	RecordFailedAudits(ctx context.Context, failedNodes []*pb.Node) (err error)
+}
+
+// Reporter records audit reports in statdb and implements the reporter interface
+type Reporter struct {
+	statdb sdbclient.Client
+}
+
+var (
+	port   = ":7777"
+	apiKey = []byte("")
+)
```

Comment:
I wonder if there is a better way of setting the port and apikey then here? In `pointerdb` we get the APIKey: `	APIKey, ok := auth.GetAPIKey(ctx)` may be this is an optimization for later?

Reply:
Yeah you're right, there's probably a better way to do the reporter and statdb setup but I'm not totally sure how to do it.